primeqa/mrc/trainers/mrc.py

```diff
@@ -250,30 +309,35 @@ def predict(self, eval_dataset=None, eval_examples=None, ignore_keys=None):
             ignore_keys: Keys to ignore in evaluation loop.
 
         Returns:
-            Answer predictions if post-processing function was provided to constructor 
+            Answer predictions if post-processing function was provided to constructor
             at instantiation, otherwise an empty dict.
         """
         eval_dataset = self.eval_dataset if eval_dataset is None else eval_dataset
         eval_dataloader = self.get_eval_dataloader(eval_dataset)
         eval_examples = self.eval_examples if eval_examples is None else eval_examples
 
-        
-        eval_loop = self.prediction_loop if self.args.use_legacy_prediction_loop else self.evaluation_loop
+        eval_loop = (
+            self.prediction_loop
+            if self.args.use_legacy_prediction_loop
+            else self.evaluation_loop
+        )
         try:
             output = eval_loop(
                 eval_dataloader,
                 description="Evaluation",
                 # No point gathering the predictions if there are no metrics, otherwise we defer to
                 # self.args.prediction_loss_only
```

Comment:
pls remove commented code.

Reply:
This is not commented out code but It is a code comment which might be useful later for new contributors. I'm inclined to keep it.